packages/prisma/schema.prisma

```diff
@@ -55,6 +55,8 @@ model Host {
   @@index([eventTypeId])
 }
 
+
+
```

Comment:
Did this to bust DB cache by CI. Somehow something wrong was cached failing the tests.

Can remove it after the PR merge .